nixos/modules/services/web-apps/libretranslate.nix

```diff
@@ -0,0 +1,203 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.libretranslate;
+  ltmanageKeysCli = pkgs.writeShellScriptBin "ltmanage-keys" ''
+    set -a
+    export HOME="/var/lib/libretranslate"
+    sudo=exec
+    if [[ "$USER" != ${cfg.user} ]]; then
+      sudo='exec /run/wrappers/bin/sudo -u ${cfg.user} --preserve-env'
+    fi
+    $sudo ${cfg.package}/bin/ltmanage keys --api-keys-db-path ${cfg.dataDir}/db/api_keys.db "$@"
+  '';
+
+in {
```

Comment:
There should be `meta.maintainers` and `meta.doc`. NixOS test would be great to have too.

Reply:
meta.doc is optionally

Reply:
Making a test is problematic, Internet access is required to download the language models.